README.md

```diff
@@ -36,6 +37,29 @@ or you can choose configuring AWS discovery using [Discovery SPI](http://docs.ha
 Starting with hazelcast-aws-1.2, it's strongly recommended to use Discovery SPI implementation as
 the former one will be deprecated.
 
+### Zone Aware Support
+
+Supported Hazelcast client version is `>= 3.8.6`
```

Comment:
Is this only for ZONE_AWARE support or in general hazelcast-client is fixed in AWS environment? cc: @sancar

Reply:
Not to break the consistency with other documents:
"This support is available for Hazelcast Client 3.8.6 and newer releases.